src/components/Chat/Chat.md

```diff
@@ -9,7 +9,7 @@ import {
   MessageList,
   MessageInput,
   Thread,
-} from '../components';
+} from '../../components';
```

Comment:
I'd prefer we not utilize index files as much as possible which is what's making this possible but whatever